parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -89,7 +89,6 @@ downloadAzureCNI() {
     CNI_TGZ_TMP=${VNET_CNI_PLUGINS_URL##*/} # Use bash builtin ## to remove all chars ("*") up to the final "/"
     retrycmd_get_tarball 120 5 "$CNI_DOWNLOADS_DIR/${CNI_TGZ_TMP}" ${VNET_CNI_PLUGINS_URL} || exit $ERR_CNI_DOWNLOAD_TIMEOUT
 }
-{{- if NeedsContainerd}}
```

Comment:
why were this function in a template block before (and those below too i guess)?

Reply:
reduced size of cloud init payload

we used to run into a lot of issues with that in aks-engine days

ironically, me making all these changes to move them into vhd will eliminate that forever, but aks engine supported vanilla ubuntu an other stuff too where that wasn't an option